doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -1116,18 +1119,19 @@ Displaying informations about scopes
    useful to know how a subterm locally occurring in the scope ofscope is
    interpreted.
 
-.. cmdv:: Print Scope scope
-
-   This displays all the notations defined in interpretation scopescope.
-   It also displays the delimiting key if any and the class to which the
-   scope is bound, if any.
-
-.. cmdv:: Print Scopes
+.. cmd:: Print Scopes
 
    This displays all the notations, delimiting keys and corresponding
    class of all the existing interpretation scopes. It also displays the
    lonely notations.
 
+.. cmdv:: Print Scope scope
```

Comment:
If you don't mind, add three more spaces in front of this whole block (then the variant will appear more clearly as a variant).

Reply:
And BTW you can use `@scope` in the `cmdv` to make it clickable and displayed differently.